examples/convert_ieeg_to_bids.py

```diff
@@ -124,12 +125,31 @@
 print("The channel coordinates before writing into BIDS: ")
 pprint([x for x in zip(ch_names, pos)])
 
+###############################################################################
+# BIDS vs MNE-Python Coordinate Systems
+# -------------------------------------
+#
+# BIDS has many acceptable coordinate systems for iEEG, which can be viewed in
+# the `specification. <https://bids-specification.readthedocs.io/en/stable/99-appendices/08-coordinate-systems.html>`_  # noqa
```

Comment:
```suggestion
# `appendix VIII`_ of the BIDS specification.
```

Add to bottom of file:

`.. _appendix VIII: https://bids-specification.readthedocs.io/en/stable/99-appendices/08-coordinate-systems.html`